upgrade-notes/storefront_20240911_211132.md

```diff
@@ -0,0 +1,4 @@
+#### Implement store features ([#3413](https://github.com/shopsys/shopsys/pull/3413))
+
+- <your changes, ie:>
```

Comment:
add some upgrade notes, please

Reply:
What upgrade notes would you imagine?

Reply:
if you have done something extraordinary then describe it. Otherwise, you can just leave  `- see #project-base-diff to update your project` and delete `- <your changes, ie:>`